app/cdap/components/GridTable/utils.tsx

```diff
@@ -47,30 +48,32 @@ export const checkFrequentlyOccuredValues = (
   gridData: IExecuteAPIResponse | undefined,
   key: string
 ) => {
-  const valueOfKey = gridData?.values.map((el) => el[key]);
-  let mostFrequentItem: number = 1;
-  let mostFrequentItemCount: number = 0;
-  let mostFrequentItemValue: string = '';
-  const mostFrequentDataItem = {
-    name: '',
-    count: 0,
-  };
-  if (Array.isArray(valueOfKey) && valueOfKey.length) {
-    valueOfKey.map((item, index) => {
-      valueOfKey.map((value, valueIndex) => {
-        if (item == value) {
-          mostFrequentItemCount++;
-        }
-        if (mostFrequentItem < mostFrequentItemCount) {
-          mostFrequentItem = mostFrequentItemCount;
-          mostFrequentItemValue = item;
-        }
+  if (gridData?.values && Array.isArray(gridData?.values)) {
```

Comment:
what if if condition failes, we should handle the return value

Reply:
added return {name:"No Data Found",count:0};

Reply:
Or shall I add {name:"Nil",count:0}